cypress/plugins/docker/index.ts

```diff
@@ -36,12 +36,16 @@ export async function dockerRun(opts: {
     const params = opts.params ?? [];
 
     if (params?.includes("-v") && userInfo.uid >= 0) {
-        // On *nix we run the docker container as our uid:gid otherwise cleaning it up its media_store can be difficult
-        params.push("-u", `${userInfo.uid}:${userInfo.gid}`);
-
         if (await isPodman()) {
-            // keep the user ID if the docker command is actually podman
-            params.push("--userns=keep-id");
+            // In podman, run as root in the container, so we're the current
+            // user on the host
```

Comment:
```suggestion
            // In podman, run as root in the container, which maps to the current
            // user on the host
```

I'm not a podman user, but my understanding that this is only true in some deployments? https://docs.podman.io/en/latest/markdown/podman-run.1.html#user-u-user-group says it only applies "when a user namespace is in use" - do we need to arrange for that to be the case?

Reply:
Yeah you're right. This is for rootless podman only, where user namespaces are always in use. I'll add a comment.